app/assets/templates/file_list.html

```diff
@@ -21,7 +21,10 @@
             <a ng-if="file.filetype === 'bruse/url'" ng-href="{{file.url}}" target="_blank">
               {{file.name}}
             </a>
-            <span ng-if="file.filetype != 'bruse/url'" ng-click="download(file.identity.id, file)">
+            <a ng-if="file.filetype.indexOf('google-apps') > -1" ng-href="{{file.link}}" target="_blank">
+              {{file.name}}
+            </a>
+            <span ng-if="file.filetype != 'bruse/url' && file.filetype.indexOf('google-apps') < 0" ng-click="download(file.identity.id, file)">
```

Comment:
Is it ok to have a long if? I tried a switch to make the code shorter, but it didn't work as I wanted it to.